onedriveb-upload

```diff
@@ -59,13 +59,19 @@ if [ -z "$1" ]; then
         exit 1
 fi
 
+curl_discover_uri
```

Comment:
We need this function only once, 'cause `api_uri` and `api_resource_id` never change.